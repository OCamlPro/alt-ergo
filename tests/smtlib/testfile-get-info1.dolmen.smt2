(set-logic ALL)

(declare-const x Int)

(assert (= (* x x) 4))

(check-sat)
(get-info :all-statistics)
(get-info :assertion-stack-levels)
(get-info :authors)
(get-info :error-behavior)
(get-info :name)
(get-info :reason-unknown)
(get-info :version)