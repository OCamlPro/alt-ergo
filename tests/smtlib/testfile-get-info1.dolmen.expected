
unknown
(
 :steps 11)

unsupported

(:authors "Alt-Ergo developers")
(:error-behavior immediate-exit)
(:name "Alt-Ergo")
(:reason-unknown incomplete)
(:version dev)
